eth/api.go

```diff
@@ -346,6 +347,40 @@ func (api *PrivateDebugAPI) GetBadBlocks(ctx context.Context) ([]*BadBlockArgs,
 	return results, nil
 }
 
+type AccountRangeResult struct {
+	result []common.Address `json:"result"`
+}
+
+func accountRange(st state.Trie, start *common.Address, maxResult int) (AccountRangeResult, error) {
```

Comment:
Question: will this overlap? If I ask for ten results, starting from `0x00..0`, and then take the last of those (address `X`), and ask for ten more -- will address `X` be included in that second set too? 

Would be good to avoid that, if so

Reply:
Sounds good.